### PR TITLE
Remove possibly useless atoms

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

**Comment by Polymorphe57** [comment]
wrong target

File: src/reactivity.ts [hunk 1]
```diff
@@ -12,18 +12,26 @@ const SOURCE = Symbol("source");
 const OBSERVER = Symbol("observer");
 const KEYS = Symbol("keys");
 const ROOT = Symbol("root");
+const SEED = Symbol("Seed");
+const MAYBE_NOT_A_SEED = Symbol("maybeNotASeed");
 
-export function atom(source: any, observer: Observer) {
+export function atom(source: any, observer: Observer, maybeNotASeed?: any) {
```
Comment (Polymorphe57) [inline src/reactivity.ts:18]: the idea is to document only the first two parameters

File: src/reactivity.ts [hunk 2]
```diff
@@ -81,15 +97,14 @@ function createAtom(source: Source, observer: Observer): Atom {
             if (!atoms.has(key)) {
               atoms.set(key, new ObserverSet());
             }
-            atoms.get(key)!.add(self);
+            atoms.get(key)!.add(newAtom);
             keys.add(key);
           }
           //
-          return atom(value, observer);
+          return atom(value, observer, MAYBE_NOT_A_SEED);
```
Comment (Polymorphe57) [inline src/reactivity.ts:104]: we can access a value that has been observed by another way: a seed can already exist on that value

File: src/reactivity.ts (hunk 1 above)
Comment (Polymorphe57) [reply src/reactivity.ts:18]: and that parameter should be not usable from outside the system

**Review by ged-odoo — APPROVED** [review]

**Pull request merged**